example/fenl/fenl_impl.hpp

```diff
@@ -113,31 +110,27 @@ class ManufacturedSolution {
   //    , a( ( 1.0 / sqrt(T_zmax) - 1.0 / sqrt(T_zmin) ) / ( zmax - zmin ) )
   //    , b( 1.0 / sqrt(T_zmin) )
 
-  const double zmin ;
-  const double zmax ;
-  const double T_zmin ;
-  const double T_zmax ;
-  const double a ;
-  const double b ;
-  const double K ;
-
-  ManufacturedSolution( const double arg_zmin ,
-                        const double arg_zmax ,
-                        const double arg_T_zmin ,
-                        const double arg_T_zmax )
-    : zmin( arg_zmin )
-    , zmax( arg_zmax )
-    , T_zmin( arg_T_zmin )
-    , T_zmax( arg_T_zmax )
-    , a( ( 1.0 / sqrt(T_zmax) - 1.0 / sqrt(T_zmin) ) / ( zmax - zmin ) )
-    , b( 1.0 / sqrt(T_zmin) )
-    , K( 1.0 / ( 6.0 * a * a ) )
-    {}
-
-  double operator()( const double z ) const
-  {
-    const double tmp = a * ( z - zmin ) + b ;
-    return 1.0 / ( tmp * tmp );
+  const double zmin;
+  const double zmax;
+  const double T_zmin;
+  const double T_zmax;
+  const double a;
+  const double b;
+  const double K;
+
+  ManufacturedSolution(const double arg_zmin, const double arg_zmax,
```

Comment:
Why is the fenl example changed in this PR?

Reply:
It uses Crs class members and clang-format decided to make it pretty during the refactor I will revert it.